pkg/lightning/backend/remote/util.go

```diff
@@ -0,0 +1,89 @@
+package remote
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/pingcap/tidb/pkg/lightning/backend"
+	"github.com/pingcap/tidb/pkg/lightning/common"
+	"github.com/pingcap/tidb/pkg/lightning/log"
+	"go.uber.org/zap"
+)
+
+const (
+	maxDuplicateBatchSize = 4 << 20
+	taskExitsMsg          = "task exists"
+
+	sleepDuration = 3 * time.Second
+	retryCount    = 6000 // 3s * 6000 = 5h
+
+	updateFlushedChunkDuration = 10 * time.Second
+)
+
+func genLoadDataTaskID(keyspaceID uint32, taskID int64, cfg *backend.EngineConfig) string {
```

Comment:
"Does this all look like util functions? Can it be merged into a single file with `util.go`?"